### ISSUE TITLE
[Frontend][Publisher] Handle long overflowing dimension names

### ISSUE THREAD

**Issue by mxosman** [Recidiviz/justice-counts#468]

**Comment by lilidworkin** [comment]
@mxosman  thanks for filing! Where is this happening, can you add a screenshot?

**Comment by mxosman** [comment]
Some dimensions are just really long - yesterday, I was trying to shorten them and add an ellipsis - but I need to do more changes to the metric config container so I decided to hold off until all of Ivan's changes are in. 

I'm a little bit back and forth with this one because on one hand it's nice to see the entire dimension name (I can easily fix the padding and spacing between it and the 'Action Required') - but, it wraps which will make the dimension box taller. It looks fine to me - but not sure if we want all the heights of the dimension boxes to be the same.

Current overflow:
<img width="659" alt="Screenshot 2023-03-07 at 1 53 26 PM" src="https://user-images.githubusercontent.com/59492998/223537503-880346b0-0a70-4ef9-9b69-00cb24543523.png">

Adjustment to spacing without ellipsis:
<img width="667" alt="Screenshot 2023-03-07 at 1 58 02 PM" src="https://user-images.githubusercontent.com/59492998/223538481-a2754338-9963-492e-b39a-de09dff5e1a2.png">

Ideally with ellipsis:
<img width="506" alt="Screenshot 2023-03-07 at 2 00 31 PM" src="https://user-images.githubusercontent.com/59492998/223538989-d5897873-d4e1-4ce6-a71f-6cc7690f8e62.png">

The ellipsis treatment here is a bit hard to do - I needed to set a fixed width for each dimension name and that doesn't play nicely when resizing and there's some jerkiness when you hover over some of them. Will need to play with more containers to see if I can get it to work well without all the side effects mentioned.
